tests/test_marker.py

```diff
@@ -88,6 +88,40 @@ def test_equality(self):
         self.assertNotEqual(m, bo)
         self.assertNotEqual(bo, m)
 
+    def test_string(self):
+        tr = otio.opentime.TimeRange(
+            otio.opentime.RationalTime(5, 24),
+            otio.opentime.RationalTime(10, 24)
+        )
+        m = otio.schema.Marker(
+            name="marker_1",
+            marked_range=tr,
+            color=otio.schema.MarkerColor.GREEN,
+            metadata={'foo': 'bar'}
+        )
+
+        self.assertEqual(
+            repr(m),
+            "otio.schema.Marker("
+            "name='marker_1', "
+            "color=u'GREEN', "
```

Comment:
It looks like the Python 3 tests are failing because there's a subtle difference in how unicode is handled between Python 2 and 3. You can try just removing the `u` from this line to see if that avoids the issue.

Reply:
@reinecke do you have any advice about how to make this work in both Python 2 and 3? Marker colors will likely stay as plain ASCII, but metadata should allow for unicode characters.

Reply:
This might help illuminate the different handling of str/unicode in Python 2 vs 3: https://portingguide.readthedocs.io/en/latest/strings.html

Reply:
My current guess on this is that it has to do with the marshaling behavior in pybind 11: https://pybind11.readthedocs.io/en/stable/advanced/cast/strings.html
Given that, it appears all strings should become unicode upon transfer back into python.

So, for python 2, it seems like we should at least be able to rely on the `u` being added consistently? Unfortunately I can't think of a clean way of branching the literals we compare based on the two python versions.

Would it make sense to just include two copies of the `repr` unittests (one marked skip in python3 and one marked skip in python2)? Then when python2 support is dropped we can remove the tests marked skip in python3 and delete the skip annotations.

An alternate approach could be to have a dedicated custom assertion that when run in python2 compares against a "patched" version of the string using something like `comp_string.replace("u'", "'").replace('u"', '"')`, but that feels pretty sketchy. The upside is that it would be a no-op in the python3 use case and the assertion could be find and replaced upon dropping python 2 support.

For reference, here is a good way to mark the unittests:
```
IS_PYTHON_2 = (sys.version_info < (3, 0))

@unittest.skipIf(IS_PYTHON_2, "In python2 string values become u-strings.")
def test_repr():
    ...do stuff...

@unittest.skipIf(not IS_PYTHON_2, "In python2 string values become u-strings.")
def test_repr_py2():
    ...do stuff...
```